app/helpers/needs_helper.rb

```diff
@@ -22,6 +22,21 @@ def needs
      'Staying Social', 'Prescription pickups', 'Book drops and entertainment', 'Dog walking']
   end
 
+  def note_category_display(note_category_id)
+    note_category = needs_note_categories.find { |category| category[:id] == note_category_id }
```

Comment:
Doesn't rails have built in enums that we can use for this instead? e.g. `enum category: { 'Successful call': 'phone_success', 'Left message': 'phone_message', ... }` (on the model) Or is this something different?

Reply:
I thought enums in the model mapped to integers in the database, but I've just tried what you suggested and it seems to work - nice!